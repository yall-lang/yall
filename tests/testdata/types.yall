(func add::i32 (a::i32 b::i32)
	(add a b))
