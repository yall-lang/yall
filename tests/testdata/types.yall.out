[
    Expression {
        kind: Item,
        values: [
            Identifier(
                "func",
            ),
            Identifier(
                "add",
            ),
            Expression(
                Expression {
                    kind: Item,
                    values: [
                        Identifier(
                            "a",
                        ),
                        Identifier(
                            "b",
                        ),
                    ],
                },
            ),
            Expression(
                Expression {
                    kind: Item,
                    values: [
                        Identifier(
                            "add",
                        ),
                        Identifier(
                            "a",
                        ),
                        Identifier(
                            "b",
                        ),
                    ],
                },
            ),
        ],
    },
]
